eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.js

```diff
@@ -85,119 +88,128 @@ export const UnwrappedMainNavigation = props => {
 
   const previewUrl = `${config.REACT_APP_LAUNCH_URL}/${
     (questionnaire || {}).id
-    }`;
+  }`;
+
+  const emptyQCode =
+    flattenedAnswers &&
+    find(flattenedAnswers, obj => obj.qCode === "" || obj.qCode === null);
```

Comment:
```suggestion
const emptyQCode = flattenedAnswers?.find(({type, qCode}) => type !== "Checkbox" && !qCode);
```

This fixes issue where Checkbox qCodes were coming up as missing (for the parent question) on imported LCS.